wire/msgmixconfirm_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2023 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package wire
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/decred/dcrd/chaincfg/chainhash"
+)
+
+func TestMixCMWire(t *testing.T) {
```

Comment:
Probably should make this match the name of the message that it's testing.  `TestMixConfirmWire`.